### ISSUE TITLE
Spigot/Bukkit

### ISSUE THREAD

**Issue by Time6628** [Microsoft/vsminecraft#19]
Will there ever be something similar to this for spigot/bukkit plugins?

**Comment by yuuka-miya** [comment]
Probably not, seeing as bukkit is all but dead.

IMO it would be better to have maven/gradle support in VS...

**Comment by sharwell** [comment]
I'm not familiar with that particular environment. Assuming it's a Java-based environment that supports plugins, it should be possible. Most of the code for the Minecraft extension is not actually specific to Minecraft, it just happened to work well in that scenario when we decided to try it one day. :+1: 

@luacs1998 improving the project system is very important. My personal opinion is NetBeans does the best overall job of supporting Maven in their project system, because it knows how to open and work with the **pom.xml** without making any other changes to it. While I don't think we'll be able to get all the way to that point within Visual Studio, we should definitely be able to iron out many of the wrinkles still in there.

**Comment by Time6628** [comment]
I'd love to see something like this for the spigot and bukkit Apis. I know many people still run cauldron servers(the forge + bukkit port).

**Comment by BrainSlugs83** [comment]
I agree.  It would be very cool to be able to create mods in Visual Studio using the popular server-side frameworks as well.
